src/Game.php

```diff
@@ -7,6 +7,10 @@
 use Monolog\Logger;
 
 use LotGD\Core\Models\Character;
+use LotGD\Core\Exceptions\ {
+    SceneNotFoundException,
+    ActionNotException
+};
```

Comment:
This is due to PSR2 predating PHP7 :) That's why I'm asking (and yeah, I have been talking about PSR12 draft). I don't care either, but we should be consequent in this.